README.md

```diff
@@ -1,14 +1,61 @@
\ No newline at end of file
+To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-busy-indicator` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
+
+## <a name="troubleshooting"></a>Troubleshooting ##
+### Path too long exception
+If you are facing a path too long exception when building this example project, close Visual Studio rename the repository to short, and build the project.
+
+## Features and Benefits
+
+### Animation types
+The [.NET MAUI Busy Indicator](https://www.syncfusion.com/maui-controls/maui-busy-indicator?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples) includes 3 pre-built animations that can be shown within the application.
+
+### Circular material
+The [.NET MAUI Busy Indicator](https://www.syncfusion.com/maui-controls/maui-busy-indicator?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples) renders an Android-style material animation in a circular path.
+
+### Linear material 
+The .NET MAUI Busy Indicator renders an Android-style material animation in a fixed horizontal line.
+
+### Cupertino
+The .NET MAUI Busy Indicator renders an iOS-style animation.
+
+### Position
+The title can be placed either at the top or bottom of the indicator animation of the .NET MAUI Busy Indicator.
+
+## Related links
+[Learn More about .NET MAUI Busy Indicator](https://www.syncfusion.com/maui-controls/maui-busy-indicator?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples)
+
+[Documentation](https://help.syncfusion.com/maui/busy-indicator/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-busy-indicator?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples)
+
+[Online example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/BusyIndicator/SampleBrowser.Maui.BusyIndicator/Samples/BusyIndicator?utm_source=github&utm_medium=listing&utm_campaign=maui-busy-indicator-github-samples)

```

Comment:
Use the below link
https://github.com/syncfusion/maui-demos/tree/master/MAUI/BusyIndicator

Reply:
Link used as commented.